docs/gallery/howto/autogen/parallel.py

```diff
@@ -1,149 +1,226 @@
 """
-=======================
-Run tasks in parallel
-=======================
+============================
+How to run tasks in parallel
+============================
 """
+
 # %%
 # Introduction
 # ============
-# In this tutorial, you will learn how to run task in parallel.
-#
-# Load the AiiDA profile.
-#
+# Once you have developed a correct and functioning workflow, the next step is often to scale it up for large datasets.
```

Comment:
The phrase "scale it up for large datasets" may be misleading in this context. The use case to scale workflow to many independent data points (e.g., run relax for multiple structures), which would typically require submitting each workgraph for each data point. Because each data point is not related.

This how-to focuses on the use case: where users run multiple tasks within the same workflow (e.g., k-point convergence tests for a single structure with different k-points). Consider rephrasing to better reflect this.

Reply:
Feel free to change this after the merge. We need to merge to reshuffle today later on